lib/cli.coffee

```diff
@@ -346,17 +345,18 @@ switch command
 
       # query DB for domains of site slugs
       getSiteDomains = (sites, done) ->
-        SLUGS = for site in sites
```

Comment:
kinda liked the old way better tbh. or you could do a `map` and then a `filter (slug) -> slug`. not a fan of coffeescript loop magic :eyes:

Reply:
addressed in following line